src/realm/sync/client.cpp

```diff
@@ -916,8 +922,8 @@ void SessionImpl::process_pending_flx_bootstrap()
                         batch_state, pending_batch.changesets.size());
 
         history.integrate_server_changesets(
-            *pending_batch.progress, downloadable_bytes, pending_batch.changesets, new_version, batch_state, logger,
-            transact, [&](const TransactionRef& tr, util::Span<Changeset> changesets_applied) {
+            *pending_batch.progress, 1.0, pending_batch.changesets, new_version, batch_state, logger, transact,
```

Comment:
Is this hardcoded to `1.0` so the progress doesn't change (e.g., go back to 0) while the bootstrap messages are being integrated?

Reply:
No, it wouldn't change regardless. This was always a hardcoded value, but it used to be a hardcoded value declared sufficiently far from the point of use that it wasn't obviously such.